source/server/BUILD

```diff
@@ -271,73 +270,10 @@ envoy_cc_library(
 )
 
 envoy_cc_library(
-    name = "api_listener_lib",
+    name = "listener_manager_lib",
     srcs = [
         "api_listener_impl.cc",
-    ],
-    hdrs = [
-        "api_listener_impl.h",
-    ],
-    deps = [
-        ":drain_manager_lib",
-        ":filter_chain_manager_lib",
-        ":listener_manager_impl",
-        "//include/envoy/network:connection_interface",
-        "//include/envoy/server:api_listener_interface",
-        "//include/envoy/server:filter_config_interface",
-        "//include/envoy/server:listener_manager_interface",
-        "//source/common/common:empty_string",
-        "//source/common/http:conn_manager_lib",
-        "//source/common/init:manager_lib",
-        "//source/common/network:resolver_lib",
-        "//source/common/stream_info:stream_info_lib",
-        "//source/extensions/filters/network/http_connection_manager:config",
-        "@envoy_api//envoy/api/v2:pkg_cc_proto",
-        "@envoy_api//envoy/api/v2/listener:pkg_cc_proto",
-        "@envoy_api//envoy/config/core/v3:pkg_cc_proto",
-        "@envoy_api//envoy/config/listener/v3:pkg_cc_proto",
-    ],
-)
-
-envoy_cc_library(
-    name = "listener_lib",
-    srcs = [
         "listener_impl.cc",
-    ],
-    hdrs = [
-        "listener_impl.h",
-    ],
-    deps = [
-        ":configuration_lib",
-        ":drain_manager_lib",
-        ":filter_chain_manager_lib",
-        ":listener_manager_impl",
-        ":transport_socket_config_lib",
-        ":well_known_names_lib",
-        "//include/envoy/server:active_udp_listener_config_interface",
-        "//include/envoy/server:filter_config_interface",
-        "//include/envoy/server:listener_manager_interface",
-        "//include/envoy/server:transport_socket_config_interface",
-        "//source/common/config:utility_lib",
-        "//source/common/init:manager_lib",
-        "//source/common/network:connection_balancer_lib",
-        "//source/common/network:listen_socket_lib",
-        "//source/common/network:socket_option_factory_lib",
-        "//source/common/network:utility_lib",
-        "//source/common/protobuf:utility_lib",
-        "//source/extensions/filters/listener:well_known_names",
-        "//source/extensions/transport_sockets:well_known_names",
-        "@envoy_api//envoy/config/core/v3:pkg_cc_proto",
-        "@envoy_api//envoy/config/listener/v3:pkg_cc_proto",
-    ],
-)
-
-# TODO(junr03): actually separate this lib from the listener and api listener lib.
```

Comment:
Can you preserve this comment so that I can come back to this?

Reply:
Whoops -- definitely.

Reply:
Done.